src/log.cpp

```diff
@@ -268,6 +268,10 @@ void set_log_level(const char* new_level) {
   }
 }
 
+void set_log_level(int new_level) {
```

Comment:
need a docstring.
Have you checked that this is callabled from Python ? I've had SWIG complain about function overloading before